src/components/pod/__snapshots__/pod.spec.js.snap

```diff
@@ -909,25 +794,17 @@ exports[`ActionButtons StyledUndoButton when isHovered prop is set should match
   color: var(--colorsActionMajorYang100);
 }
 
-.c35 .c2 {
+.c28.c28 .c2 {
+  top: -2px;
+  height: 16px;
+  width: 16px;
   color: var(--colorsActionMajor500);
 }
 
-.c35:hover .c2 {
+.c28.c28 .c2 {
   color: var(--colorsActionMajorYang100);
 }
 
-.c35 .c2 {
-  margin-left: 0px;
-  margin-right: var(--spacing100);
-  margin-bottom: 0px;
-  height: 16px;
-}
-
-.c35 .c2 svg {
-  margin-top: 0;
-}
-
```

Comment:
Maybe it's a good time to remove this snapshot? What do you think about that?

Reply:
I would be happy to look into this - I agree the tests we have for `Pod` would be better as explicit tests rather than snapshots

Reply:
I already removed the snapshot in a later commit part of this PR 😄